hydra-redis/classes_objects.py

```diff
@@ -0,0 +1,118 @@
+import urllib.request as ur
+import json
+from redisgraph import Node, Edge
+
+
+class Classes:
+    """Contains all the classes endpoint and the objects"""
+
+    def __init__(self, redis_graph):
+        self.redis_graph = redis_graph
+
+    def objects_property(self, objects_node, new_list, url_node, api_doc):
```

Comment:
Explain what this function is doing. Also use a more precise Docstring. `Makes nodes for those properties which have properties in itself.` this is not very clear.